index.d.ts

```diff
@@ -384,5 +384,13 @@ export declare enum HTTP_STATUS {
  * Convert the numeric status code to its appropriate title.
  * @param statusCode One of the available status codes in this package
  * @returns {String} The associated title of the passed status code
+ * @throws {Error} The status code does not exist
  */
 export declare function getStatusText(statusCode: number): string;
+
+/**
+ * Convert the status code title to its appropriate numeric value
```

Comment:
I have some problem with the word "title" in these comments. We should refer to them consistently as "status code" and "status text". 

In fact the IETF spec calls it a "Reason Phrase" so ideally we could change all of the comments from "title" -> "reason phrase".

```
The reason-phrase element exists for the sole purpose of providing a
   textual description associated with the numeric status code, mostly
   out of deference to earlier Internet application protocols that were
   more frequently used with interactive text clients.  A client SHOULD
   ignore the reason-phrase content.
```